teamA/jeongHaeMan/.env.sample

```
@@ -0,0 +1,11 @@
+DATABASE_URL="mysql://root:toddlf0704@127.0.0.1:3306/westagram"
```

제 3자 혹은 외부에서 해당 URL을 통해 database에 접근할 수 있을 것 같네요.

개인 정보는 지우고 키워드로 대체해주세요!
```suggestion
DATABASE_URL="rdbms://username:password@host:portnumber/db-name"
```